docs/content/developers/release-management.md

```diff
@@ -40,7 +40,7 @@ The following “Repository secret” environment variables must be added to <ht
 
 ### Prerelease Tasks
 
-* Make sure [`version-history.md`](https://github.com/drud/ddev/blob/master/version-history.md) is up to date.
+* Make sure [`version-history.md`](https://github.com/drud/ddev/blob/master/version-history.md) is up-to-date.
```

Comment:
This “up to date” doesn’t need to be hyphenated:

```suggestion
* Make sure [`version-history.md`](https://github.com/drud/ddev/blob/master/version-history.md) is up to date.
```

If it was modifying a subject, then the hyphen would apply. (_Simon’s “up-to-date tie” was not as culturally relevant as he thought._ vs. _Simon’s tie, it turns out, was not up to date._)